cluster/examples/kubernetes/ceph/csi/example/rbd/storageclass.yaml

```diff
@@ -21,8 +21,8 @@ parameters:
     # The secrets have to contain Ceph admin credentials.
     csi.storage.k8s.io/provisioner-secret-name: csi-rbd-secret
     csi.storage.k8s.io/provisioner-secret-namespace: default
-    csi.storage.k8s.io/node-publish-secret-name: csi-rbd-secret
-    csi.storage.k8s.io/node-publish-secret-namespace: default
+    csi.storage.k8s.io/node-stage-secret-name: csi-rbd-secret
```

Comment:
What is the difference between the publish and stage secrets?

Reply:
Earlier CSI was doing Nodepublish and unpublish volume which requires mon IP  and user ID and password to map rbd volume to a node, with new changes we will be mapping rbd device during node stage instead of publishing